### PR TITLE
OpenRCON Window

### PR DESCRIPTION
**Pull request by Bletch1971**
1. Added functionality to store the previous values used to open the RCONWindow and load then when this load re-opens.

NOTE: Only saved when the user clicks the Connect button. If they exit out, the values are NOT stored.

### CODE REVIEW

File: ARK Server Manager/Lib/ServerRCON.cs [hunk 1]
```diff
@@ -308,6 +308,10 @@ private void HandleCommand(ConsoleCommand command)
 
                 if (this.Players.Count == 0 || newPlayerList.Count > 0)
                 {
+                    var message = $"Update Player Details requested.";
```
Comment (ChronosWS) [inline ARK Server Manager/Lib/ServerRCON.cs]: The $ prefix is unnecessary here.

File: ARK Server Manager/OpenRCON.xaml [hunk 2]
```diff
@@ -22,10 +22,10 @@
         <Label Grid.Row="0" Grid.Column="0" Content="Server IP:"/>
         <TextBox Grid.Row="0" Grid.Column="1" Text="{Binding ServerIP}" TabIndex="0"/>
         
-        <Label Grid.Row="1" Grid.Column="0" Content="Server Port:"/>
-        <TextBox Grid.Row="1" Grid.Column="1" Text="{Binding ServerPort}" TabIndex="1"/>
+        <Label Grid.Row="1" Grid.Column="0" Content="RCON Port:"/>
```
Comment (ChronosWS) [inline ARK Server Manager/OpenRCON.xaml]: Looks like we missed some Localization strings.  Do you want to change these to be bound to strings in the localization file?

File: ARK Server Manager/Lib/ServerRCON.cs (hunk 1 above)
Comment (Bletch1971) [inline ARK Server Manager/Lib/ServerRCON.cs]: Oh shit, that is what you get when you copy/paste.

File: ARK Server Manager/OpenRCON.xaml (hunk 2 above)
Comment (Bletch1971) [inline ARK Server Manager/OpenRCON.xaml]: I will add this window to the localisation file.

**Comment by Bletch1971** [comment]
Have gone through all the forms and converted them over to use the localisation files.

**Comment by Bletch1971** [comment]
We still need to go through the code to find messages etc and convert those, but not yet please!

**Pull request merged**